### PR TITLE
Fix some warnings

### PR DESCRIPTION
**Pull request by Holzhaus**

### CODE REVIEW

File: src/controllers/hid/hidcontrollerpreset.h [hunk 1]
```diff
@@ -15,7 +15,7 @@ class HidControllerPreset : public ControllerPreset {
 
     bool savePreset(const QString& fileName) const override;
 
-    virtual void accept(ControllerPresetVisitor* visitor);
-    virtual void accept(ConstControllerPresetVisitor* visitor) const;
-    virtual bool isMappable() const;
+    virtual void accept(ControllerPresetVisitor* visitor) override;
```
Comment (daschuer) [inline src/controllers/hid/hidcontrollerpreset.h:18]: override implies virtual. That's why we remove virtual here.

**Review by daschuer — CHANGES_REQUESTED** [review]
Thank you for taking care.

File: src/controllers/hid/hidcontrollerpreset.h (hunk 1 above)
Comment (Holzhaus) [reply src/controllers/hid/hidcontrollerpreset.h]: Done.

**Comment by uklotzde** [comment]
LGTM. @daschuer?

**Review by daschuer — APPROVED** [review]

**Comment by daschuer** [comment]
LGTM Thank you.

**Pull request merged**